pkg/framework/coremodel/staticregistry/provide.go

```diff
@@ -3,15 +3,28 @@ package staticregistry
 import (
 	"github.com/grafana/grafana/pkg/coremodel/dashboard"
 	"github.com/grafana/grafana/pkg/framework/coremodel"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
 )
 
 // ProvideRegistry provides a simple static Registry for coremodels.
 // Coremodels have to be manually added.
 // TODO dynamism
 func ProvideRegistry(
+	feat *featuremgmt.FeatureManager,
 	dashboard *dashboard.Coremodel,
 ) (*coremodel.Registry, error) {
-	return coremodel.NewRegistry(
+	cmlist := []coremodel.Interface{
 		dashboard,
-	)
+	}
+
+	// return coremodel.NewRegistry(cmlist...)
+	// Uncomment above line and everything delete below, as well as unfeatured.go,
+	// once coremodels are no longer feature flagged
```

Comment:
I am surprised that the flag lives here -- I would expect that flag to be about rather we want dashboards to be checked

Reply:
See comment about potentially large number of feature flags